ui.apps/src/main/content/jcr_root/conf/venia/settings/wcm/templates/product-page/structure/.content.xml

```diff
@@ -38,6 +38,8 @@
                     skipNavigationRoot="true"
                     sling:resourceType="venia/components/structure/navigation/v1/navigation"
                     structureDepth="2"/>
+                <minicart jcr:primaryType="nt:unstructured"
```

Comment:
Please make sure to add minicart to all templates. Currently category-page, content-page, landing-page and search-results doesn't have it.